.github/workflows/ci.yml

```diff
@@ -37,7 +37,4 @@ jobs:
         env:
           CI: true
       - if: matrix.os == 'ubuntu-latest'
-        continue-on-error: true
-        uses: codecov/codecov-action@v1.0.0
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
The new version of codecov action does not require a token, so `CODECOV_TOKEN` secret can be removed